GenderPayGap.WebUI/wwwroot/scripts/gov_uk_front_end.js

```diff
@@ -2647,30 +2647,22 @@
 	Tabs.prototype.setAttributes = function ($tab) {
 		// set tab attributes
 		var panelId = this.getHref($tab).slice(1);
-		$tab.setAttribute('id', 'tab_' + panelId);
-		$tab.setAttribute('role', 'tab');
 		$tab.setAttribute('aria-controls', panelId);
 		$tab.setAttribute('aria-selected', 'false');
-		$tab.setAttribute('tabindex', '-1');
 
 		// set panel attributes
 		var $panel = this.getPanel($tab);
-		$panel.setAttribute('role', 'tabpanel');

```

Comment:
Remind me why we're removing this one? I _think_ it's because it's already being set anyway but I just want to be sure.

Reply:
Yes, the JS isn't working properly (which is in another ticket) so we removed this attribute so it's clear that's it's only being set in the views with C#/Html.